war/pom.xml

```diff
@@ -417,83 +425,92 @@ THE SOFTWARE.
                   <version>6.2.0-3</version>
                   <type>hpi</type>
                 </artifactItem>
 
                 <artifactItem>
-                  <!-- dependency of junit and workflow-api -->
+                  <!-- dependency of checks-api, junit, workflow-api, and workflow-support -->
                   <groupId>org.jenkins-ci.plugins.workflow</groupId>
                   <artifactId>workflow-step-api</artifactId>
                   <version>639.v6eca_cd8c04a_a_</version>
                   <type>hpi</type>
                 </artifactItem>
                 <artifactItem>
-                  <!-- dependency of workflow-api -->
+                  <!-- dependency of workflow-api and workflow-support -->
                   <groupId>org.jenkins-ci.plugins</groupId>
                   <artifactId>scm-api</artifactId>
                   <version>621.vda_a_b_055e58f7</version>
                   <type>hpi</type>
                 </artifactItem>
                 <artifactItem>
-                  <!-- dependency of junit and scm-api -->
+                  <!-- dependency of scm-api and workflow-step-api -->
                   <groupId>org.jenkins-ci.plugins</groupId>
                   <artifactId>structs</artifactId>
                   <version>324.va_f5d6774f3a_d</version>
                   <type>hpi</type>
                 </artifactItem>
                 <artifactItem>
+                  <!-- detached after 2.16 -->
                   <groupId>org.jenkins-ci.plugins</groupId>
                   <artifactId>bouncycastle-api</artifactId>
                   <version>2.26</version>
                   <type>hpi</type>
                 </artifactItem>
                 <artifactItem>
+                  <!-- detached after 2.86 -->
                   <groupId>org.jenkins-ci.plugins</groupId>
                   <artifactId>command-launcher</artifactId>
                   <version>90.v669d7ccb_7c31</version>
                   <type>hpi</type>
                 </artifactItem>
                 <artifactItem>
+                  <!-- detached after 2.112 -->
                   <groupId>org.jenkins-ci.plugins</groupId>
                   <artifactId>jdk-tool</artifactId>
                   <version>63.v62d2fd4b_4793</version>
                   <type>hpi</type>
                 </artifactItem>
                 <artifactItem>
+                  <!-- detached after 2.163 -->
                   <groupId>io.jenkins.plugins</groupId>
                   <artifactId>jaxb</artifactId>
                   <version>2.3.7-1</version>
                   <type>hpi</type>
                 </artifactItem>
                 <artifactItem>
+                  <!-- detached after 2.281 -->
                   <groupId>org.jenkins-ci.modules</groupId>
                   <artifactId>sshd</artifactId>
-                  <version>3.236.ved5e1b_cb_50b_2</version>
+                  <version>3.249.v2dc2ea_416e33</version>
```

Comment:
Another benefit of this PR is that it pulls in https://github.com/jenkinsci/sshd-plugin/pull/88, which consumes `instance-identity` as a plugin rather than a module. This is technically more correct, since this version of core does not deliver `instance-identity` as a module.